### DISCUSSION TITLE
Congrats on the v3.1 release!

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#2565] [❤️]
I'm excited to start using reflines in plots.

**Comment by frosencrantz** [comment]
Agreed. Congratulations on the new release!  Thank you for all your efforts @saulpw, @anjakefala, @midichef, and @thejud!  Also, thank you @reagle and @ajkerrigan for your guide contributions.  I'm sure there were many other helpers, too.  I'm very happy to see multiple of my wishlist items get some closure!
